MITRE ATT&CK for Azure Sentinel.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
<p>This is a monster cell, maybe consider having a bit more details in Markdown about what its doing, and have an output of some sort so its clearer that it has done as expected.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>That's the biggest cell I've ever seen!</p><p>If it's all just utility functions that are executed and forgotten it's OK but if it's actually doing something procedural, I agree about the comments and/or break it up.</p>